src/base/settingsstorage.cpp

```diff
@@ -26,7 +26,9 @@
  * but you are not obligated to do so. If you do not wish to do so, delete this
  * exception statement from your version.
  */
+#include "settingsstorage.h"
```

Comment:
...so give it back as it was.
If you haven't noticed, inkludes in new code goes in a certain order and visually divided into groups.